zppy/templates/ts.bash

```diff
@@ -147,19 +147,39 @@ EOF
   cp -s $dest/*_{{ '%04d' % (yr_start) }}??_{{ '%04d' % (yr_end) }}??.nc $input_dir
   dest_cmip={{ output }}/post/{{ component }}/{{ grid }}/cmip_ts/{{ frequency }}
   mkdir -p ${dest_cmip}
+
   {{ e3sm_to_cmip_environment_commands }}
+
```

Comment:
Yes, this needs to be rebased off the refactored code to use the e3sm_to_cmip task.